Automattic/jetpack
# Search: Show correct count for paginated results

Fixes #19656.

#### Changes proposed in this Pull Request:
* Shows correct count for paginated results.
  * Prior to this change, we would only rely on the number of results stored in our Redux state due to `response.total` being unreliable in the past (p8oabR-DS-p2#comment-5144).
  * With this change, we only override `response.total` if it's less than the number of results stored in our Redux state.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Apply this change to your site with a Jetpack Search subscription. Ensure that Instant Search is enabled.
* Perform a search that yields more than 10 results. Ensure that the page says "Found n results" where n is the total count of all matching results.
